src/utils/logging/prepareResponse.ts

```diff
@@ -8,9 +8,24 @@ import { parseBody } from '../request/parseBody'
 export function prepareResponse(res: SerializedResponse<any>) {
   const responseHeaders = objectToHeaders(res.headers)
 
+  // Parse a response JSON body for preview in the logs
+  let parsedBody: ReturnType<typeof parseBody> | Promise<any> = parseBody(
+    res.body,
+    responseHeaders,
+  )
+
+  if (parsedBody instanceof ReadableStream) {
+    const response = new Response(parsedBody, { headers: responseHeaders })
```

Comment:
Alas, this won't work since `prepareResponse` is called in _any_ environment, including Node. The current version of MSW still promises support for Node 16, which doesn't have the Fetch API available globally. This will break for those consumers. 

We have to depend on something different, it seems.

Reply:
Yeah :/ that's sad

Reply:
Actually, I just checked and this `prepareResponse` pipeline is only run in the browser. We should be safe relying on the `Response` constructor here, given we don't get any type errors in general. 

Testing this locally now...